packages/toolpad-components/src/Image.tsx

```diff
@@ -12,6 +12,8 @@ export interface ImageProps {
   fit: 'contain' | 'cover' | 'fill' | 'none' | 'scale-down';
 }
 
+const DEFAULT_IMG_SRC = 'https://mui.com/static/branding/mui-x/Mocktable-dark.png';
```

Comment:
i found this image a bit dark and hard to understand - maybe we can use something bigger and more photographic?

Reply:
Something from https://unsplash.com/ potentially?

Reply:
those seem nice!

Reply:
How about an image of [ El Teide ](https://unsplash.com/photos/S12z98cqAzM)? :D